app/assets/stylesheets/new_design/icons.scss

```diff
@@ -24,4 +24,24 @@ i {
   &.folder {
     background-image: image-url("icons/folder.svg");
   }
+
+  &.accept {
+    background-image: image-url("icons/accept.svg");
+  }
+
+  &.close {
+    background-image: image-url("icons/close.svg");
+  }
+
+  &.without-continuation {
+    background-image: image-url("icons/without-continuation.svg");
+  }
+
+  &.edit {
+    background-image: image-url("icons/edit-folder-blue.svg");
```

Comment:
Pinaillage : qu'est-ce qui justifie le fait qu'on specifie la couleur dans le nom de fichier ? (idem pour `in-progress-blue`) ?

Reco : s'assurer que classe et nom de fichier sont identiques ?

Reply:
ok on veillera à uniformiser